### ISSUE TITLE
[Bug]: PackageInfo returns 1.0.0 with no build number on Windows

### ISSUE THREAD

**Issue by seikosantana** [fluttercommunity/plus_plugins#929]
### Platform

Windows

### Plugin

package_info_plus

### Version

1.4.2

### Flutter SDK

3.0.1

### Steps to reproduce

1. Ensure bindings initialized in entry point method `WidgetsFlutterBinding.ensureInitialized();`
2. Load the package information
```dart
PackageInfo? loadedPackageInfo;

Future loadPackageInfo() async {
  loadedPackageInfo ??= await PackageInfo.fromPlatform();
}
```
3. Get the result in a widget state
```dart
@override
  void initState() {
    super.initState();
    initAll();
  }

  void initAll() async {
    await loadPackageInfo();
    setState(() {
      packageInfo = loadedPackageInfo;
    });
    await performAuthorization();
  }
```
4. Display the package information on build
```dart
Text(
                "${packageInfo?.version ?? ""}.${packageInfo?.buildNumber ?? ""}"),
```
5. See 1.0.0. in the UI

### Logs

```shell
[  +77 ms] executing: [C:\fluttersdk\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +585 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms] fb57da5f945d02ef4f98dfd9409a72b7cce74268
[   +1 ms] executing: [C:\fluttersdk\flutter/] git tag --points-at fb57da5f945d02ef4f98dfd9409a72b7cce74268
[  +90 ms] Exit code 0 from: git tag --points-at fb57da5f945d02ef4f98dfd9409a72b7cce74268
[   +1 ms] 3.0.1
[   +9 ms] executing: [C:\fluttersdk\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +72 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms] origin/stable
[        ] executing: [C:\fluttersdk\flutter/] git ls-remote --get-url origin
[  +64 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +156 ms] executing: [C:\fluttersdk\flutter/] git rev-parse --abbrev-ref HEAD
[  +72 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +93 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +70 ms] executing: C:\Users\Seiko Santana\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[ +117 ms] List of devices attached
[  +18 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[  +25 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +208 ms] Multiple devices found:
[ +105 ms] Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19044.1766]
[   +4 ms] Chrome (web)      • chrome  • web-javascript • Google Chrome 96.0.4664.45
[   +1 ms] Edge (web)        • edge    • web-javascript • Microsoft Edge 103.0.1264.37
[   +2 ms] [1]: Windows (windows)
[   +1 ms] [2]: Chrome (chrome)
[   +1 ms] [3]: Edge (edge)
[   +5 ms] Please choose one (To quit, press "q/Q")
[   +1 ms] : 
[+9971 ms] 1
[  +17 ms] Skipping pub get: version match.
[  +37 ms] Found plugin esptouch_smartconfig at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\esptouch_smartconfig-1.1.0\
[  +31 ms] Found plugin mobile_scanner at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.1.1\
[   +7 ms] Found plugin package_info_plus at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus-1.4.2\
[   +2 ms] Found plugin package_info_plus_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_linux-1.0.5\
[   +4 ms] Found plugin package_info_plus_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_macos-1.3.0\
[   +5 ms] Found plugin package_info_plus_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_web-1.0.5\
[   +5 ms] Found plugin package_info_plus_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_windows-1.0.5\
[   +6 ms] Found plugin path_provider_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.6\
[   +5 ms] Found plugin path_provider_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.0.6\
[  +10 ms] Found plugin shared_preferences at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.15\
[   +2 ms] Found plugin shared_preferences_android at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.12\
[   +2 ms] Found plugin shared_preferences_ios at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_ios-2.1.1\
[   +3 ms] Found plugin shared_preferences_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.1\
[   +2 ms] Found plugin shared_preferences_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_macos-2.0.4\
[   +3 ms] Found plugin shared_preferences_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.4\
[   +2 ms] Found plugin shared_preferences_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.1\
[  +18 ms] Found plugin wifi_iot at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_iot-0.3.15+3\
[   +3 ms] Found plugin wifi_scan at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_scan-0.3.0\
[  +56 ms] Found plugin esptouch_smartconfig at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\esptouch_smartconfig-1.1.0\
[  +13 ms] Found plugin mobile_scanner at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.1.1\
[   +1 ms] Found plugin package_info_plus at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus-1.4.2\
[   +4 ms] Found plugin package_info_plus_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_linux-1.0.5\
[   +1 ms] Found plugin package_info_plus_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_macos-1.3.0\
[   +1 ms] Found plugin package_info_plus_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_web-1.0.5\
[   +1 ms] Found plugin package_info_plus_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_windows-1.0.5\
[   +1 ms] Found plugin path_provider_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.6\
[   +2 ms] Found plugin path_provider_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.0.6\
[   +4 ms] Found plugin shared_preferences at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.15\
[   +1 ms] Found plugin shared_preferences_android at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.12\
[   +1 ms] Found plugin shared_preferences_ios at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_ios-2.1.1\
[   +1 ms] Found plugin shared_preferences_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.1\
[   +1 ms] Found plugin shared_preferences_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_macos-2.0.4\
[   +2 ms] Found plugin shared_preferences_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.4\
[   +1 ms] Found plugin shared_preferences_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.1\
[  +14 ms] Found plugin wifi_iot at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_iot-0.3.15+3\
[   +2 ms] Found plugin wifi_scan at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_scan-0.3.0\
[  +66 ms] Generating C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +127 ms] Initializing file store
[  +15 ms] Skipping target: gen_localizations
[   +6 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents: C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\package_config_subset,C:\Users\Seiko
Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\flutter_build\dart_plugin_registrant.dart}
[  +20 ms] Found plugin esptouch_smartconfig at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\esptouch_smartconfig-1.1.0\
[  +12 ms] Found plugin mobile_scanner at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.1.1\
[   +2 ms] Found plugin package_info_plus at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus-1.4.2\
[   +2 ms] Found plugin package_info_plus_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_linux-1.0.5\
[   +1 ms] Found plugin package_info_plus_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_macos-1.3.0\
[   +3 ms] Found plugin package_info_plus_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_web-1.0.5\
[   +2 ms] Found plugin package_info_plus_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_windows-1.0.5\
[   +2 ms] Found plugin path_provider_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.6\
[   +2 ms] Found plugin path_provider_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.0.6\
[   +3 ms] Found plugin shared_preferences at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.15\
[   +2 ms] Found plugin shared_preferences_android at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.12\
[   +1 ms] Found plugin shared_preferences_ios at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_ios-2.1.1\
[   +1 ms] Found plugin shared_preferences_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.1\
[   +2 ms] Found plugin shared_preferences_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_macos-2.0.4\
[   +3 ms] Found plugin shared_preferences_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.4\
[   +1 ms] Found plugin shared_preferences_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.1\
[   +8 ms] Found plugin wifi_iot at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_iot-0.3.15+3\
[   +2 ms] Found plugin wifi_scan at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_scan-0.3.0\
[   +9 ms] gen_dart_plugin_registrant: Complete
[   +3 ms] Skipping target: _composite
[   +3 ms] complete
[   +9 ms] Launching lib\main.dart on Windows in debug mode...
[   +6 ms] C:\fluttersdk\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\fluttersdk\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root C:\fluttersdk\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/
--incremental --target=flutter --debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill C:\Users\SEIKOS~1\AppData\Local\Temp\flutter_tools.ff7dda7a\flutter_tool.dded6a7d\app.dill --packages C:\Users\Seiko
Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill
build\c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --source C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\flutter_build\dart_plugin_registrant.dart --source package:flutter/src/dart_plugin_registrant.dart
-Dflutter.dart_plugin_registrant=file:///C:/Users/Seiko%20Santana/AndroidStudioProjects/iotaboard-frontend/.dart_tool/flutter_build/dart_plugin_registrant.dart --enable-experiment=alternative-invalidation-strategy
[  +36 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[  +63 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[   +1 ms] [
             {
               "instanceId": "5a3e66e8",
               "installDate": "2021-06-16T05:35:13Z",
               "installationName": "VisualStudio/16.11.13+32413.511",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise",
               "installationVersion": "16.11.32413.511",
               "productId": "Microsoft.VisualStudio.Product.Enterprise",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Enterprise 2019",
               "description": "Scalable, end-to-end solution for teams of any size",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "installedChannelUri": "https://aka.ms/vs/16/release/channel",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-v16.11#16.11.13",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2022-04-27T02:44:14.3530237Z",
               "catalog": {
                 "buildBranch": "d16.11",
                 "buildVersion": "16.11.32413.511",
                 "id": "VisualStudio/16.11.13+32413.511",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.11.13",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "13",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.11.13+32413.511",
                 "requiredEngineVersion": "2.11.69.53063"
               },
               "properties": {
                 "campaignId": "1422098050.1623821307",
                 "channelManifestId": "VisualStudio.16.Release/16.11.13+32413.511",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +2 ms] Building Windows application...
[  +20 ms] <- compile package:iotaboard/main.dart
[   +3 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\windows -B build\windows -G Visual Studio 16 2019
[  +75 ms] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
[  +73 ms] -- Configuring done
[ +216 ms] -- Generating done
[  +19 ms] -- Build files have been written to: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows
[   +8 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug --target INSTALL --verbose
[ +215 ms] Microsoft (R) Build Engine version 16.11.2+f32259642 for .NET Framework
[   +1 ms] Copyright (C) Microsoft Corporation. All rights reserved.
[  +69 ms] Build started 6/25/2022 2:32:09 PM.
[ +333 ms] Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[   +1 ms] Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +40 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +46 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[        ] Done Building Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +188 ms] Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[   +2 ms] Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) on node 1
(default targets).
[   +1 ms] Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) is building "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\flutter_assemble.vcxproj"
(5) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +3 ms] CustomBuild:
[   +1 ms]   "The build of 'C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\CMakeFiles\2284b66eae0cda47a7940204fd5ca6d2\flutter_assemble.rule' depends on 'C:\USERS\SEIKO
SANTANA\ANDROIDSTUDIOPROJECTS\IOTABOARD-FRONTEND\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of 'C:\Users\Seiko
Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\CMakeFiles\d9521f2d358819cd5f60cee5b67dbf2b\flutter_windows.dll.rule'. The items cannot be built in parallel."
[ +109 ms]   Generating C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/flutter_windows.dll, C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/flutter_export.h, C:/Users/Seiko
Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/flutter_windows.h, C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/flutter_messenger.h, C:/Users/Seiko
Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/flutter_plugin_registrar.h, C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/flutter_texture_registrar.h, C:/Users/Seiko
Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc, C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc, C:/Users/Seiko
Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc, C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc, C:/Users/Seiko
Santana/AndroidStudioProjects/iotaboard-frontend/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+3102 ms]   [ +123 ms] executing: [C:\fluttersdk\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +741 ms]   [ +779 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms]   [        ] fb57da5f945d02ef4f98dfd9409a72b7cce74268
[        ]   [   +1 ms] executing: [C:\fluttersdk\flutter/] git tag --points-at fb57da5f945d02ef4f98dfd9409a72b7cce74268
[ +172 ms]   [ +172 ms] Exit code 0 from: git tag --points-at fb57da5f945d02ef4f98dfd9409a72b7cce74268
[   +1 ms]   [        ] 3.0.1
[  +12 ms]   [  +15 ms] executing: [C:\fluttersdk\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +100 ms]   [  +99 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms]   [        ] origin/stable
[   +1 ms]   [        ] executing: [C:\fluttersdk\flutter/] git ls-remote --get-url origin
[ +107 ms]   [ +109 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms]   [        ] https://github.com/flutter/flutter.git
[ +123 ms]   [ +124 ms] executing: [C:\fluttersdk\flutter/] git rev-parse --abbrev-ref HEAD
[ +107 ms]   [ +107 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms]   [        ] stable
[ +201 ms]   [ +201 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +7 ms]   [  +11 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +296 ms]   [ +304 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +4 ms]   [  +12 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +5 ms]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +5 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[ +123 ms]   [ +146 ms] Initializing file store
[  +65 ms]   [  +65 ms] Skipping target: gen_localizations
[  +54 ms]   [  +54 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents: C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\package_config_subset}
[  +12 ms]   [  +13 ms] unpack_windows: Starting due to {}
[ +936 ms]   [ +935 ms] unpack_windows: Complete
[+4926 ms]   [+4927 ms] Found plugin esptouch_smartconfig at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\esptouch_smartconfig-1.1.0\
[  +22 ms]   [  +22 ms] Found plugin mobile_scanner at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\mobile_scanner-1.1.1\
[   +5 ms]   [   +5 ms] Found plugin package_info_plus at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus-1.4.2\
[   +2 ms]   [   +2 ms] Found plugin package_info_plus_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_linux-1.0.5\
[   +8 ms]   [   +9 ms] Found plugin package_info_plus_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_macos-1.3.0\
[   +3 ms]   [   +3 ms] Found plugin package_info_plus_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_web-1.0.5\
[   +3 ms]   [   +2 ms] Found plugin package_info_plus_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\package_info_plus_windows-1.0.5\
[   +6 ms]   [   +6 ms] Found plugin path_provider_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-2.1.6\
[   +5 ms]   [   +4 ms] Found plugin path_provider_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-2.0.6\
[   +6 ms]   [   +7 ms] Found plugin shared_preferences at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-2.0.15\
[   +2 ms]   [   +2 ms] Found plugin shared_preferences_android at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_android-2.0.12\
[   +2 ms]   [   +2 ms] Found plugin shared_preferences_ios at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_ios-2.1.1\
[   +2 ms]   [   +2 ms] Found plugin shared_preferences_linux at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-2.1.1\
[   +6 ms]   [   +5 ms] Found plugin shared_preferences_macos at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_macos-2.0.4\
[   +4 ms]   [   +5 ms] Found plugin shared_preferences_web at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-2.0.4\
[  +13 ms]   [  +13 ms] Found plugin shared_preferences_windows at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-2.1.1\
[  +19 ms]   [  +19 ms] Found plugin wifi_iot at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_iot-0.3.15+3\
[   +2 ms]   [   +2 ms] Found plugin wifi_scan at C:\fluttersdk\flutter\.pub-cache\hosted\pub.dartlang.org\wifi_scan-0.3.0\
[  +48 ms]   [  +49 ms] gen_dart_plugin_registrant: Complete
[   +6 ms]   [   +6 ms] kernel_snapshot: Starting due to {}
[  +17 ms]   [  +17 ms] C:\fluttersdk\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\fluttersdk\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root
C:\fluttersdk\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --target=flutter --no-print-incremental-dependencies -DFLUTTER_WEB_AUTO_DETECT=true -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --packages
C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\package_config.json --output-dill C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\flutter_build\18a18205b0b9a7dc17d41483392b4832\app.dill --depfile C:\Users\Seiko
Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\flutter_build\18a18205b0b9a7dc17d41483392b4832\kernel_snapshot.d --source C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\.dart_tool\flutter_build\dart_plugin_registrant.dart --source
package:flutter/src/dart_plugin_registrant.dart -Dflutter.dart_plugin_registrant=file:///C:/Users/Seiko%20Santana/AndroidStudioProjects/iotaboard-frontend/.dart_tool/flutter_build/dart_plugin_registrant.dart package:iotaboard/main.dart
[+27108 ms]   [+27107 ms] kernel_snapshot: Complete
[+1234 ms]   [+1235 ms] debug_bundle_windows_assets: Starting due to {}
[ +216 ms]   [ +215 ms] debug_bundle_windows_assets: Complete
[+1204 ms]   [+1204 ms] Persisting file store
[  +34 ms]   [  +34 ms] Done persisting file store
[  +34 ms]   [  +34 ms] build succeeded.
[  +29 ms]   [  +29 ms] "flutter assemble" took 36,575ms.
[ +159 ms]   [ +159 ms] ensureAnalyticsSent: 145ms
[   +2 ms]   [   +2 ms] Running shutdown hooks
[   +1 ms]   [        ] Shutdown hooks complete
[   +1 ms]   [   +1 ms] exiting with code 0
[ +178 ms] FinalizeBuildStatus:
[   +2 ms]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[  +13 ms] InitializeBuildStatus:
[   +1 ms]   Creating "flutter_wrapper_app.dir\Debug\flutter_.421707FE.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +3 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[ +121 ms] ClCompile:
[   +2 ms]   All outputs are up-to-date.
[  +50 ms] Lib:
[   +3 ms]   All outputs are up-to-date.
[   +2 ms]   flutter_wrapper_app.vcxproj -> C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\Debug\flutter_wrapper_app.lib
[  +28 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.421707FE.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "flutter_wrapper_app.dir\Debug\flutter_.421707FE.tlog\flutter_wrapper_app.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[  +99 ms] Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (6) on node 1
(default targets).
[   +3 ms] InitializeBuildStatus:
[   +1 ms]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.3DBD94C3.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +18 ms] ClCompile:
[   +1 ms]   All outputs are up-to-date.
[   +8 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[   +1 ms]   flutter_wrapper_plugin.vcxproj -> C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[  +12 ms] FinalizeBuildStatus:
[   +2 ms]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.3DBD94C3.tlog\unsuccessfulbuild".
[   +4 ms]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.3DBD94C3.tlog\flutter_wrapper_plugin.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[  +86 ms] Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\runner\iotaboard.vcxproj" (7) on node 1 (default
targets).
[   +2 ms] InitializeBuildStatus:
[   +1 ms]   Creating "iotaboard.dir\Debug\iotaboard.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +62 ms] ClCompile:
[   +2 ms]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\CL.exe /c /I"C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\windows" /I"C:\Users\Seiko
Santana\AndroidStudioProjects\iotaboard-frontend\windows\flutter\ephemeral" /I"C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\windows\flutter\ephemeral\cpp_client_wrapper\include" /Zi /nologo /W4 /WX /diagnostics:column /Od /Ob0 /D WIN32 /D _WINDOWS
/D _HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"iotaboard.dir\Debug\\"
/Fd"iotaboard.dir\Debug\vc142.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\windows\flutter\generated_plugin_registrant.cc" "C:\Users\Seiko
Santana\AndroidStudioProjects\iotaboard-frontend\windows\runner\flutter_window.cpp"
[ +213 ms]   generated_plugin_registrant.cc
[ +657 ms]   flutter_window.cpp
[ +900 ms]   Generating Code...
[ +129 ms] ResourceCompile:
[   +1 ms]   All outputs are up-to-date.
[  +45 ms] Link:
[   +1 ms]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\runner\Debug\iotaboard.exe"
/INCREMENTAL /ILK:"iotaboard.dir\Debug\iotaboard.ilk" /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\windows\flutter\ephemeral\flutter_windows.dll.lib" kernel32.lib user32.lib gdi32.lib winspool.lib
shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /manifestinput:"C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\windows\runner\runner.exe.manifest"
/DEBUG /PDB:"C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/iotaboard.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:/Users/Seiko
Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/iotaboard.lib" /MACHINE:X64  /machine:x64 iotaboard.dir\Debug\Runner.res
[   +1 ms]   iotaboard.dir\Debug\flutter_window.obj
[        ]   iotaboard.dir\Debug\main.obj
[   +1 ms]   iotaboard.dir\Debug\utils.obj
[        ]   iotaboard.dir\Debug\win32_window.obj
[        ]   iotaboard.dir\Debug\generated_plugin_registrant.obj
[ +307 ms]   iotaboard.vcxproj -> C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\runner\Debug\iotaboard.exe
[   +8 ms] FinalizeBuildStatus:
[        ]   Deleting file "iotaboard.dir\Debug\iotaboard.tlog\unsuccessfulbuild".
[        ]   Touching "iotaboard.dir\Debug\iotaboard.tlog\iotaboard.lastbuildstate".
[        ] Done Building Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\runner\iotaboard.vcxproj" (default targets).
[   +7 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[   +8 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[        ] Done Building Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +7 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +93 ms] PostBuildEvent:
[   +1 ms]   setlocal
[   +1 ms]   "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake
[   +1 ms]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[  +63 ms]   -- Install configuration: "Debug"
[  +30 ms]   -- Up-to-date: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/icudtl.dat
[   +1 ms]   -- Up-to-date: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/flutter_windows.dll
[  +12 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets
[   +1 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +3 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[   +2 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/fonts
[   +1 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +8 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[ +571 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/lottie_files
[   +3 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/lottie_files/40101-waiting-pigeon.json
[   +4 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/lottie_files/58014-pink-pigeon-shruging-hands.json
[   +4 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/lottie_files/629-empty-box.json
[   +5 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/lottie_files/82418-searching-animation.json
[   +5 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/lottie_files/no-connection.json
[   +4 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/lottie_files/not-found.json
[   +5 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/NOTICES
[  +12 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/NOTICES.Z
[   +6 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/packages
[   +3 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[   +2 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[   +2 ms]   -- Installing: C:/Users/Seiko Santana/AndroidStudioProjects/iotaboard-frontend/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[  +36 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[   +1 ms] Done Building Project "C:\Users\Seiko Santana\AndroidStudioProjects\iotaboard-frontend\build\windows\INSTALL.vcxproj" (default targets).
[   +1 ms] Build succeeded.
[   +1 ms]     0 Warning(s)
[   +1 ms]     0 Error(s)
[   +1 ms] Time Elapsed 00:00:46.07
[  +58 ms] Building Windows application... (completed in 46.8s)
[ +462 ms] Observatory URL on device: http://127.0.0.1:24613/lQ9RYulhyao=/
[  +14 ms] Caching compiled dill
[  +87 ms] Connecting to service protocol: http://127.0.0.1:24613/lQ9RYulhyao=/
[ +313 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:24613/lQ9RYulhyao=/.
[ +186 ms] DDS is listening at http://127.0.0.1:24616/6Fw1EZKCeyY=/.
[ +163 ms] Successfully connected to service protocol: http://127.0.0.1:24613/lQ9RYulhyao=/
[  +53 ms] DevFS: Creating new filesystem on the device (null)
[  +33 ms] DevFS: Created new filesystem on the device (file:///C:/Users/SEIKOS~1/AppData/Local/Temp/iotaboard-frontenddf641cad/iotaboard-frontend/)
[   +3 ms] Updating assets
[ +168 ms] Syncing files to device Windows...
[   +3 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +1 ms] Processing bundle.
[   +2 ms] <- recompile package:iotaboard/main.dart b4809620-b4ab-44e7-912b-455b748c867e
[        ] <- b4809620-b4ab-44e7-912b-455b748c867e
[   +4 ms] Bundle processing done.
[ +173 ms] flutter: perform auth
[  +66 ms] flutter: token c2Vpa29zYW50YW5hOmlvdGF0ZWNoLWFkbWlu
[  +40 ms] Updating files.
[   +1 ms] DevFS: Sync finished
[   +2 ms] Syncing files to device Windows... (completed in 294ms)
[   +1 ms] Synced 0.0MB.
[   +3 ms] <- accept
[   +4 ms] Connected to _flutterView/0x201670f85a0.
[   +4 ms] Flutter run key commands.
[   +2 ms] r Hot reload. 
[   +1 ms] R Hot restart.
[   +2 ms] h List all available interactive commands.
[   +1 ms] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ]  Running with sound null safety 
[   +1 ms] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:24616/6Fw1EZKCeyY=/
[ +475 ms] The Flutter DevTools debugger and profiler on Windows is available at: http://127.0.0.1:9100?uri=http://127.0.0.1:24616/6Fw1EZKCeyY=/
```
```


### Flutter Doctor

```shell
[✓] Flutter (Channel stable, 3.0.1, on Microsoft Windows [Version 10.0.19044.1766], locale en-US)
    • Flutter version 3.0.1 at C:\fluttersdk\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision fb57da5f94 (5 weeks ago), 2022-05-19 15:50:29 -0700
    • Engine revision caaafc5604
    • Dart version 2.17.1
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\Seiko Santana\AppData\Local\Android\sdk
    • Platform android-31, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[✓] Visual Studio - develop for Windows (Visual Studio Enterprise 2019 16.11.13)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise
    • Visual Studio Enterprise 2019 version 16.11.32413.511
    • Windows 10 SDK version 10.0.19041.0

[✓] Android Studio (version 2021.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[✓] VS Code, 64-bit edition (version 1.68.1)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.42.0

[✓] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19044.1766]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 96.0.4664.45
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 103.0.1264.37

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
```

**Comment by sitaryo** [comment]
you can change the "VERSION_AS_STRING" and "VERSION_AS_NUMBER" in windows/runner/Runner.rc 
manually. if you want to change it automatically, the feature will be rls in next stable release of Flutter.
see https://github.com/flutter/flutter/issues/73652#issuecomment-1185955595

**Comment by iqfareez** [comment]
I run the app against Flutter 3.3 and the version number matches the `pusbpec.yaml` file. I think this issue can be closed.

https://docs.flutter.dev/deployment/windows#updating-the-apps-version-number

https://docs.flutter.dev/development/platform-integration/windows/version-migration

**Comment by seikosantana** [comment]
Probably unrelated but on android I always have 1 as revision number even if in pubspec i wrote it to be 0

**Comment by miquelbeltran** [comment]
@seikosantana yes it is unrelated to this issue, I believe someone else has reported it already in another ticket.

Thanks @sitaryo and @iqfareez for the heads up!
